docs: add 26.1.1.22 server release notes (#47919)
ydb-platform/ydb: ydb/docs/en/core/changelog-server.md

## Version 26.1 {#26-1}

### Version 26.1.1.22 {#26-1-1-22}

Release date: July 27, 2026.

#### Bug Fixes

> * [Fixed](https://github.com/ydb-platform/ydb/pull/46894) Kafka API Proxy authentication for local users: with `DomainLoginOnly`, users from the root domain could not access tenant databases.

Reviewer: ```suggestion
* [Fixed](https://github.com/ydb-platform/ydb/pull/46894) Kafka API authentication for local users: with enabled `DomainLoginOnly` setting, users could not access tenant databases.
```
